dandi/files/bases.py

```diff
@@ -469,6 +480,21 @@ def get_validation_errors(
         else:
             # make sure that we have some basic metadata fields we require
             try:
+                # Ensure latest version of NWB Inspector is installed and used client-side
+                current_version, max_version = self._get_inspector_versions()
+                if current_version < max_version:
+                    errors.extend(
+                        [
+                            f"NWB Inspector version {current_version} is installed - please "
+                            f"use the latest release of the NWB Inspector ({max_version}) when "
+                            "performing `dandi validate`. "
+                            "To update, please run `pip install -U nwbinspector`."
```

Comment:
Note that `pip install -U` is useful but might be misleading if dandi-cli and/or nwbinspector installed through a different distribution (e.g. conda or who knows whatnot later, e.g. debian), so
```suggestion
                            "To update, run `pip install -U nwbinspector` if you installed it with `pip`."
```